contracts/src/LightClient.sol

```diff
@@ -7,7 +7,19 @@ import { IPlonkVerifier } from "./interfaces/IPlonkVerifier.sol";
 import { PlonkVerifier } from "./libraries/PlonkVerifier.sol";
 import { LightClientStateUpdateVK as VkLib } from "./libraries/LightClientStateUpdateVK.sol";
 
-/// @notice A light client for HotShot consensus. Keeping track of its finalized states in safe,
+/// @title Light Client Contract
+/// @notice This contract serves as an always-on client
+/// that verifies HotShot's state (Espresso's consensus state) which can be used by
+/// Rollup contracts on L1 (Ethereum).
+/// This state is submitted by any state-prover with evidence which is
+/// a SNARK proof that proves consensus.
+/// This contract also keeps track of the current epoch and stake table.
```

Comment:
Implicitly the HotShot state will refer to some stake table state but the light client contract does not interact nor read from the stake table contract.

Reply:
Well, my observation above is a bit confusing. I think no change is needed in the code comment. Just wanted to highlight that the light client proactively updates the epoch for other contracts to read (e.g. stake table), but only receives a commitment to the stake table state.

Reply:
So this is important because it means the light client contract MUST be updated at least once per epoch. We should document this both in the gitbook (did not check yet maybe it's already written down!) and in the comment of the function `newFinalizedState`.

This is not relevant for Decaf but we can easily forget about this implicit property. Let us document it.

Reply:
Thanks so much for highlighting this `the light client contract MUST be updated at least once per epoch` is not currently documented so i'll add to both the function comments and the gitbook

Reply:
I can also remove `and stake table` if that makes it less confusing - don't want to add it if it can cause an incorrect  assumptions

Reply:
Ok, let us remove it for now, especially because the stake table is fixed for Decaf. Thank you.

Reply:
https://github.com/EspressoSystems/espresso-sequencer/commit/dd362f7ab0db1761c763427661bdf2a2703cc8d6